tests/Integration/Core/Module/ModuleRepositoryTest.php

```diff
@@ -55,11 +56,46 @@ protected function setUp(): void
         $translator = $this->createMock(Translator::class);
         $translator->method('trans')->willReturnArgument(0);
 
+        $hookManager = $this->createMock(HookManager::class);
+        $hookExecMethodMock = function () {
+            // cf. HookManager::exec() method signature
+            list(
+                $hook_name,
+                $hook_args,
+                $id_module,
+                $array_return,
+                $check_exceptions,
+                $use_push,
+                $id_shop
+                ) = func_get_args();
```

Comment:
```suggestion
        $hookExecMethodMock = function ($hook_name, $hook_args, $id_module, $array_return, $check_exceptions, $use_push, $id_shop)
```

Why complexify this function when you can simply define its parameters matching the mocked method?